### PR TITLE
add multiple tasks support

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: docs/design/task-launch-order-within-job.md [hunk 1]
```diff
@@ -62,19 +62,45 @@ spec:
       ......
     - replicas: 1
       name: mpimaster
-      dependentTask: 
+      dependentTasks:
```
Comment (shinytang6) [inline docs/design/task-launch-order-within-job.md:65]: what about `dependson` ?

Comment (hwdef) [reply docs/design/task-launch-order-within-job.md:65]: you mean `dependsOn`?

Comment (shinytang6) [reply docs/design/task-launch-order-within-job.md:65]: yes

Comment (hwdef) [reply docs/design/task-launch-order-within-job.md:65]: ok, I agree.😄

**Review by shinytang6 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**